lib/stitches/auto_fill.py

```diff
@@ -276,6 +299,12 @@ def graph_is_valid(graph):
     return not networkx.is_empty(graph) and networkx.is_eulerian(graph)
 
 
+def graph_make_valid(graph):
+    if not networkx.is_eulerian(graph):
+        return networkx.eulerize(graph)
```

Comment:
Oh, great idea!  We're already in a weird situation with such a tiny shape, eulerize can't make it worse.

Reply:
Thanks. I thought so too.